specs/contract_signatures/01_main.rst

```diff
@@ -53,8 +53,8 @@ Diagram_
 .. code:: javascript
     
     {
-        "type": "signTypedDataResult"
-        "hash": <hex-string> // hash of the payload according to EIP712 (unsigned)
+        "type": "signTypedDataResult",
+        "hash": <hex-string>, // hash of the payload according to EIP712 (unsigned)
         "signature": <hex-string> // Signature of the hash of the payload according to EIP712

```

Comment:
Maybe we could link https://gnosis-safe.readthedocs.io/en/latest/contracts/signatures.html